python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-traffic-engineering.j2

```diff
@@ -55,6 +55,10 @@ router traffic-engineering
 {%             endfor %}
 {%         endif %}
 {%     endif %}
+{%     if router_traffic_engineering.twamp_light_sender_profile is arista.avd.defined %}
+   !
+   twamp-light sender profile {{ router_traffic_engineering.twamp_light_sender_profile }}
```

Comment:
is the config order here correct?

Reply:
I thought i had checked but seems i messed it up anyway. Should be correct now!